domenetjenester/fpsak/src/main/java/no/nav/foreldrepenger/fpsak/BehandlingRestKlientImpl.java

```diff
@@ -40,8 +52,10 @@
 import no.nav.foreldrepenger.fpsak.dto.ytelsefordeling.YtelseFordelingDto;
 import no.nav.foreldrepenger.melding.behandling.BehandlingRelLinkPayload;
 import no.nav.foreldrepenger.melding.behandling.BehandlingResourceLink;
+import no.nav.vedtak.felles.integrasjon.rest.JsonMapper;
 import no.nav.vedtak.felles.integrasjon.rest.OidcRestClient;
 import no.nav.vedtak.konfig.KonfigVerdi;
+import no.nav.vedtak.util.StringUtils;
 
 @ApplicationScoped
 public class BehandlingRestKlientImpl implements BehandlingRestKlient {
```

Comment:
@extrsi  vil ikke dette gi feil med parallelitet? Så lenge den er ApplicationScoped, vil ikke samme instants brukes for alle requester?  Kanskje request scoped?

Reply:
ApplictionScoped, gjør et instance blir brukt og den lever til applikasjon er slot av.  og ApplicatioScoped operations are threadsafe.

Reply:
Ja, men den samler jo opp json data på instansnivå?